crates/control/src/role_assignment.rs

```diff
@@ -536,6 +548,7 @@ fn decide_if_claiming_striker_or_other_role(
     player_number: PlayerNumber,
     cycle_start_time: SystemTime,
     game_controller_state: Option<&GameControllerState>,
+    optional_roles: &Vec<Role>,
```

Comment:
same as above: use a slice instead of `&Vec`

Reply:
done